src/battle_interface.c

```diff
@@ -3467,10 +3467,14 @@ void TryAddLastUsedBallItemSprites(void)
     if (gLastThrownBall == 0
       || (gLastThrownBall != 0 && !CheckBagHasItem(gLastThrownBall, 1)))
     {
+        u16 firstBall = ITEM_NONE;
         // we're out of the last used ball, so just set it to the first ball in the bag
```

Comment:
```suggestion
        u16 firstBall = ITEM_NONE;

        // we're out of the last used ball, so just set it to the first ball in the bag
```

Reply:
Although it's not needed to assign it a value since the first time you invoke `firstBall` is to assign it an entirely different one.

Reply:
True, I will remove this PR, make the changes on master and create a new PR.